tests/test_non_ascii_paths/makefile

```diff
@@ -1,7 +0,0 @@
-all:
```

Comment:
I'd accidentally duplicated it from https://github.com/mozilla/dxr/blob/master/tests/test_non_ascii_paths/code/makefile, and this file is not run.